Phonebook.Frontend/src/app/shared/dialogs/user-information/user-information.component.html

```diff
@@ -0,0 +1,24 @@
+<div>
+  <div class="center">
+    <h2 mat-dialog-title i18n="User-InformationDialog|Headline@@Headline">User Information</h2>
+    <div mat-dialog-content *ngIf="onMyProfile">
+      <span i18n="User-InformationDialog|Profile@@Your Profile">
+        Dear {{ data.Firstname }},<br />
+        unfortunately, you can't change your user information via the Phonebook interface. <br />
+        If your data is incorrect, please contact your hr department.
+      </span>
+    </div>
+    <div mat-dialog-content *ngIf="!onMyProfile">
+      <span i18n="User-InformationDialog|Profile@@Colleague Profile">
+        Dear {{ data.Firstname }}<br />
+        Is there something wrong with the profile? <br />
+        Please let your colleague know that the data is incorrect and that it can be changed through your hr department.
+      </span>
+    </div>
+  </div>
+  <div class="actions">
```

Comment:
```suggestion
  <div mat-dialog-actions class="pb-center-element">
```

Reply:
With your removed styles the dialog will look like this:
![image](https://user-images.githubusercontent.com/39745446/67389274-8fedb600-f59a-11e9-9181-ff50b9b99533.png)